kitsune/wiki/jinja2/wiki/includes/document_macros.html

```diff
@@ -12,6 +12,20 @@
   <h1 class="sumo-page-heading">{{ document.title }}</h1>
 {%- endmacro %}
 
+{% macro document_metadata(document, helpful_votes, product) -%}
+<div id="document_meta" class="has-border-bottom">
+  <span class="product">{{ product }}</span>
+  <span class="last-updated">
+    <img class="pencil" src="{{ static('/sumo/img/pencil.svg') }}" /> <strong>Last updated:</strong> <span class="time">{{ document.current_revision.created|naturaltime }}</span>
+  </span>
+  {% if helpful_votes %}
+  <span class="helpful-info">
+    <img class="thumbsup" src="{{ static('/sumo/img/thumbs-up.svg') }}"/><span class="helpful-count">{{ helpful_votes }}%</span> of users found this helpful
```

Comment:
Likewise here. This should be `webpack_static('sumo/img/thumbs-up.svg')` -- note the lack of an initial forward slash in the path -- instead of `static('/sumo/img/thumbs-up.svg')` in order to pick up the hashed version created by webpack.